bpm_projects_api/core/security.py

```diff
@@ -26,7 +26,11 @@ def get_secret_key():
     return app.config['SECRET_KEY']
 
 
-def token_required(f):
+class PolicyError(Exception):
```

Comment:
What is the main goal of this class? , Are we going to refactor it later or do we pretend to pass this type of exception?

Reply:
Decorators allows a nice programming practice [AOP](https://en.wikipedia.org/wiki/Aspect-oriented_programming), which for me is the best way of validating security. The idea of having a custom PolicyError is for managing better the errors:
- On pattern matching. E.g.